src/main/java/sirius/biz/process/ErrorContext.java

```diff
@@ -258,10 +283,11 @@ public void handleInContext(UnaryOperator<String> failureDescription, String lab
         try {
             return Optional.ofNullable(producer.create());
         } catch (HandledException exception) {
-            logException(exception);
+            logException(exception, ProcessLogType.ERROR);
```

Comment:
Use new `logError()`?
```suggestion
            logError(exception);
```